Jira87198 replace Kommander UI with DKP UI (#4244)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/security/oidc/index.md

-   To interact with Kubernetes API, usually through `kubectl`.

> -   To interact with the D2iQ DKP UI, which has GUI dashboards for Prometheus, Grafana, and so on.

Reviewer: Just getting rid of some Vale errors to have those 3 checks turn green :wink:. 
```suggestion
-   To interact with the D2iQ DKP UI, which has GUI dashboards for Prometheus, Grafana, etc.
```

In Kommander, Dex comes pre-configured with a client for these access use cases. The clients talk to Dex for authentication. Dex talks to the configured Identity Provider, or IdP, (for example LDAP, SAML and so on) to perform the actual task of authenticating the user.

> If the user authenticates successfully, Dex pulls the user’s information from the IdP and forms an OpenID token. The token contains this information and returns it to the respective client’s callback URL. The client or end user uses this token for communicating with the D2iQ DKP UI or Kubernetes API respectively.

Reviewer: ```suggestion
If the user authenticates successfully, Dex pulls the user’s information from the IdP and forms an OpenID token. The token contains this information and returns it to the respective client’s callback URL. The client or end user uses this token for communicating with the DKP UI or Kubernetes API respectively.
```
